constants.py

```diff
@@ -685,3 +686,56 @@
 
 kVelocityControlMotorType = DCMotor.falcon500()
 kVelocityControlkV = 0.01
+
+kAngleMotorRatio = (64 / 16) * (60 / 18)
+kShootingMotorRatio = 24 / 36
+
+# change numbers later
+
+kAngleMotorCANId = 99
+kAngleMotorName = "ShooterAngleMotor"
+kAngleMotorPGain = 0.1
+kAngleMotorIGain = 0
+kAngleMotorDGain = 0
+kAngleMotorInverted = False
+
+kLeftShootingMotorCANId = 999
+kLeftShootingMotorName = "LeftShootingMotor"
+kLeftShootingMotorPIDSlot = 0
+kLeftShootingMotorPGain = 1
```

Comment:
oh god this would go way too fast
do some kV action for this motor
kV is in rps / V, so take max free speed and divide by voltage for that free speed